lib/scrabble.rb

```diff
@@ -1,18 +1,35 @@
 class Scrabble
+  attr_reader :word
+
+  def initialize
+    @word = nil
+  end
 
   def score(word)
-    1
+    @word = word.upcase!
+    word = sub_points(@word)
+    word = add_points(word)
+  end
```

Comment:
Like that you're using this as a top-level method that is then calling down to supporting methods. Helps to make the overall logic/thinking clear.